astroquery/alma/core.py

```diff
@@ -249,6 +249,13 @@ def stage_data(self, uids):
         uids : list or str
             A list of valid UIDs or a single UID.
             UIDs should have the form: 'uid://A002/X391d0b/X7b'
+        expand_tarfiles : bool
+            Expand the tarfiles to obtain lists of all contained files?  If
```

Comment:
Why the '?'

Reply:
Well, it's a boolean, so I posed it as a question, apparently.
```suggestion
            Expand the tarfiles to obtain lists of all contained files.  If
```